docker/Dockerfile

```diff
@@ -223,7 +235,15 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # Install vllm wheel first, so that torch etc will be installed.
 RUN --mount=type=bind,from=build,src=/workspace/dist,target=/vllm-workspace/dist \
     --mount=type=cache,target=/root/.cache/uv \
-    uv pip install --system dist/*.whl --verbose
+    uv pip install --system dist/*.whl --verbose \
+    --extra-index-url https://download.pytorch.org/whl/cu128
```

Comment:
It seems like this should changed based on `CUDA_VERSION`, as this won't work for CUDA_VERSION=12.6

Reply:
Yeah, you're right, this needs to change depending on `CUDA_VERSION`, let me add that check to cover 11.8, 12.6, and 12.8 matching those from PyTorch 2.7.0